src/richie/apps/google_analytics/templates/richie/base.html

```diff
@@ -0,0 +1,20 @@
+{% extends "richie/base.html" %}
+{% block css_links %}
+{% comment %}The code should be added near the top of the HTML header tag, before any other script or CSS tags{% endcomment %}
+    {% if GOOGLE_ANALYTICS and GOOGLE_ANALYTICS.TRACKING_ID %}
+        <script async src="https://www.googletagmanager.com/gtag/js?id={{ GOOGLE_ANALYTICS.TRACKING_ID | safe }}"></script>
+        <script>
+            window.dataLayer = window.dataLayer || [];
+            function gtag(){dataLayer.push(arguments);}
+            gtag('js', new Date());
+            gtag('set', {'dimension1': '{{ GOOGLE_ANALYTICS.ORGANIZATIONS_CODE }}'});
+            gtag('set', {'dimension2': '{{ GOOGLE_ANALYTICS.COURSE_CODE }}'});
+            gtag('set', {'dimension3': '{{ GOOGLE_ANALYTICS.COURSE_RUN }}'});
+            gtag('set', {'dimension4': '{{ GOOGLE_ANALYTICS.COURSE_RESOURCE_LINK }}'});
+            gtag('set', {'dimension5': '{{ GOOGLE_ANALYTICS.COURSE_NAME }}'});
+            gtag('config', '{{ GOOGLE_ANALYTICS.TRACKING_ID | safe }}');
+        </script>
+    {% endif %}
```

Comment:
Why not create a dedicade block `analytics`? In this way, we could easily override only this part and it's clearer for new comers.

Reply:
Done. This makes sense when the google analytics was merged to the core application.